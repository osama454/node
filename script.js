document.body.textContent = 123;