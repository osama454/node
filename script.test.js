import { hello } from "./script";

describe("label", () => {
  it("label", () => {
    expect(hello()).toMatchSnapshot();
    expect(hello()).toBe("hello");
    expect({k:"v"}).toEqual({k:"v"});
    expect(5).toBeGreaterThan(4);
    expect(5).toBeGreaterThanOrEqual(5);
    expect(4).toBeLessThan(5);
    expect(5).toBeLessThanOrEqual(5);
  });

});
