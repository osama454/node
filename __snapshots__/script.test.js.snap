// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateFieldControl label 1`] = `"hello"`;

exports[`label label 1`] = `"hello"`;
